en/services/mission.md

```diff
@@ -105,6 +105,39 @@ Both `MISSION_REQUEST` and `MISSION_ITEM` messages are now deprecated, and shoul
 If `MISSION_REQUEST` is recieved the system should instead respond with [MISSION_ITEM_INT](#MISSION_ITEM_INT) items (as though it  received [MISSION_REQUEST_INT](#MISSION_REQUEST_INT)).
 
 
+## Frames & Positional Information
+
+By convention, mission items use `param5`, `param6`, `param7` for positional information when needed (and otherwise as "free use" parameters).
+The table below shows that the positional parameters can be local (x, y, z), global (latitude, longitude, altitude), and also the data type used to store the parameters in the `MISSION_ITEM_INT` message.
+
+param | type | Local | Global
+--- | --- | --- | --- | ---
+param5 | `int32_t` | x | Latitude
+param6 | `int32_t` | y | Longitude
+param7 | `float` | z | Altitude (global - relative or absolute)
+
+The co-ordinate frame of positional parameters is defined in the `MISSION_ITEM_INT.frame` field using a [MAV_FRAME](#MAV_FRAME) value.
+
+The global frames are prefixed with `MAV_FRAME_GLOBAL_*`. 
+Mission items should use frame variants that have the suffix `_INT`: e.g. `MAV_FRAME_GLOBAL_RELATIVE_ALT_INT`, `MAV_FRAME_GLOBAL_INT`, `MAV_FRAME_GLOBAL_TERRAIN_ALT_INT`.
+When using these frames, latitude and longitude values must be encoded by multiplying the degrees by 1E7 (e.g. the latitude 69.69000000 would be sent as 69.69000000x1E7 = 696900000).
+This convention ensures that no positional information is lost in transmission due to truncation or rounding errors.
+
+A number of local frames are also specified.
+Local frame position values that are sent in integer field parameters must be encoded as *position in meters x 1E4* (e.g. 5m would be encoded and sent as 50000).
+If sent in messages `float` parameter fields the value should be sent as-is.
+
+
+> **Note** The non-INT global frames (e.g. `MAV_FRAME_GLOBAL_RELATIVE_ALT`) are intended to be used with messages that have `float` fields for positional information (e.g. `MISSION_ITEM`).
+  Position values should be sent unencoded when using these frames (i.e. no need to multiply by 1E7).
```

Comment:
So you're saying that a local mission in meters (which is not supported by PX4, but in theory) should be using MISSION_ITEM instead of MISSION_ITEM_INT, right? But we actually deprecated using MISSION_ITEM a few months ago, so how does that work?

Reply:
@julianoes Actually no - this note only refers to *The non-INT **global** frames* - it says nothing about the local frames. 
This is essentially saying _why_ we have the non_int variants of frames - and that you shouldn't use them in missions anymore. You can still use them in command_long through.

The use of local frames is covered it the preceding paragraph - essentially if you send in an int (mission_int) I would expect the meter values to be multiplied by 1xE4; if sent in a float then sent as a float. 

Either way, we don't need MISSION_ITEM. 

Does that make sense?

Reply:
Side note: we should have never have int and long variants but just used double. Side note closed :smile: 

Oh, don't us the non-int frames, I think I got you.

Reply:
@julianoes `double` for mayor of mavlink 3!
@hamishwillee Thanks, looks great

Reply:
@ValentineStone Yes, double would be nice. Julian, is it worth introducing it or just "too late".

@julianoes Re "Don't use the non-INT *global frames*". That is exactly what I mean and should have said. So I modified it to say that. Will merge now :-)